envision/types.py

```diff
@@ -38,7 +38,7 @@ class VehicleType(str, Enum):
     Coach = "coach"
     Truck = "truck"
     Trailer = "trailer"
-    Car = "car"
+    Passenger = "passenger"
```

Comment:
This should be reverted too.  
Note that the list here in `envison/types` will not match the union of the two possible type lists that we currently support (the keys to the `VEHICLE_CONFIG` dict in `vehicle.py` and then the values of the `vehicle_type` fields in the `VehicleConfig` objects in that dict).

To fix this (in another PR), we will either need to add a function to map from the bigger list into the list of 5 things that envision supports, or (probably better, but non-trivial) unify across our vehicle typing schemes.